python/ray/experimental/state.py

```diff
@@ -137,11 +136,11 @@ def _initialize_global_state(self,
         been populated or we exceed a timeout.
 
         Args:
-            redis_ip_address: The IP address of the node that the Redis server
-                lives on.
-            redis_port: The port that the Redis server is listening on.
+            redis_address: The Redis address to connect.
             redis_password: The password of the redis server.
         """
+        redis_ip_address, redis_port = redis_address.split(":")
+        redis_port = int(redis_port)
         self.redis_client = redis.StrictRedis(
```

Comment:
Does it make sense to use `create_redis_client()` instead of `redis.StrictRedis()`?
I think it's better to unify them.